tests/conftest.py

```diff
@@ -271,3 +271,26 @@ def pytest_addoption(parser):
         default=None,
         help="The SSM parameter to use to check STS access. Optional",
     )
+    parser.addoption(
+        "--namespace",
+        default=None,
+        help="The namespace to test."
+    )
+    parser.addoption(
+        "--prod",
+        action="store_true",
+        help=("Indicates that tests are being run in the production account "
+              "and that the production cloudformation bucket ought to be used.")
+    )
+
+@pytest.fixture(scope="session")
+def namespace(pytestconfig):
+     yield pytestconfig.getoption("namespace")
+
+@pytest.fixture(scope="session")
+def artifact_bucket(pytestconfig):
```

Comment:
I'm wondering if it's safe to write test related things to our artifact bucket just because it's technically where our scripts for our etl workflow are found? Esp having the option of using the production artifact bucket. 

If this is mainly for integration testing (doing these actions in a physical bucket instead of mocking it), then would it be better for us to have a separate bucket for this use case?

Reply:
I'm not worried about it since every S3 reference uses the `workspace_key_prefix`, which is formatted like `{namespace}/tests/test_json_to_parquet`.

I'm actually going to remove the `--prod` optional flag because I don't think it's a plausible use case to run unit tests in the production environment.